src/librsync_export.h

```diff
@@ -1,6 +1,10 @@
 #ifndef LIBRSYNC_EXPORT_H
 #  define LIBRSYNC_EXPORT_H
 
+#ifdef LIBRSYNC_STATIC_DEFINE
+#  define LIBRSYNC_EXPORT
+#  define LIBRSYNC_NO_EXPORT
```

Comment:
Actually, we never use LIBRSYNC_NO_EXPORT, so you can delete this line.

This macro is created by GenerateExportHeader for cases when you explicitly want to not export something and have configured your default as "public". We set our default to "hidden" and only use LIBRSYNC_EXPORT.

Reply:
Fixed.